packages/superset-ui-chart-controls/src/operators/sortOperator.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitationsxw
+ * under the License.
+ */
+import { PostProcessingSort, RollingType } from '@superset-ui/core';
+import { PostProcessingFactory } from './types';
+import { TIME_COLUMN } from './utils';
+
+export const sortOperator: PostProcessingFactory<PostProcessingSort | undefined> = (
```

Comment:
nit thought: I instinctively feel like `PostProcessingFactory<PostProcessingSort> | undefined` may more readable than `PostProcessingFactory<PostProcessingSort | undefined>`. What do you think?

Reply:
This may not work. the interface `PostProcessingFactory` defined what is a return value from the operator.